interfaces/builtin/mir_test.go

```diff
@@ -51,24 +53,61 @@ var _ = Suite(&MirInterfaceSuite{
 	},
 })
 
+func (s *MirInterfaceSuite) SetUpTest(c *C) {
```

Comment:
It looks like this is redundant with `var _ Suite ...`. I would suggest to remove one of them or restore s.slot in TestSecComp (where it is used and presumably restored here).